WS2012R2/lisa/remote-scripts/ica/FC_disks.sh

```diff
@@ -182,6 +182,10 @@ do
         UpdateTestState $ICA_TESTFAILED
         exit 90
     fi
+
+# Check for Call traces
+CheckCallTracesWithDelay 20
```

Comment:
must source utils.sh in order to call this function

Reply:
yes, sorry, my mistake, thank you.